addons/event/tests/test_mailing.py

```diff
@@ -159,7 +159,6 @@ def test_mail_template_creation(self):
                 template.email_from,
                 "{{ (object.event_id.organizer_id.email_formatted or object.event_id.company_id.email_formatted or user.email_formatted or '') }}"
             )
-            self.assertFalse(template.email_to)
```

Comment:
Could probably save the form and check invisible values like that on saved template (email_cc and email_to should be False)